code/global/Utils.cs

```diff
@@ -138,6 +138,16 @@ public static T GetObjectByType<T>(Type type)
             return Library.Create<T>(type);
         }
 
+        /// <summary>
+        /// Returns an instance of the given type by the given type `Type`.
+        /// </summary>
+        /// <param name="type">A derived `Type` of the given type</param>
+        /// <returns>Instance of the given type object</returns>
+        public static Entity GetObjectByType(Type type)
+        {
+            return Library.Create<Entity>(type);
+        }
+
```

Comment:
Is this really needed? Regarding the function above. If so, the comment has to be adjusted regarding `Entity`

Reply:
`/// <returns>Instance of the given type object</returns>`, but it returns an Instance of an Entity

Reply:
I do not know, suggested change?

Reply:
Not needed, we have the same method defined above.